packages/grafana-schema/src/common/mudball.cue

```diff
@@ -247,4 +247,88 @@ VizTooltipOptions: {
 	sort: SortOrder
 } @cuetsy(kind="interface")
 
+Labels: {
+	[string]: string
+} @cuetsy(kind="interface")
+
+// TODO docs | generic type
+ScopedVar: {
```

Comment:
This should only ever exist in a runtime context -- I am not sure we want it to cross into schema land, or do we have a clear way to say it should never be serialized?